ql/src/test/results/clientnegative/ct_noperm_loc.q.out

```diff
@@ -1,11 +1 @@
-PREHOOK: query: create table foo0(id int) location 'hdfs://### HDFS PATH ###'
-PREHOOK: type: CREATETABLE
-PREHOOK: Input: hdfs://### HDFS PATH ###
-PREHOOK: Output: database:default
-PREHOOK: Output: default@foo0
-POSTHOOK: query: create table foo0(id int) location 'hdfs://### HDFS PATH ###'
```

Comment:
I wonder why we have lost these messages from the q.out - the q file didn't changed at all; the other tests have PREHOOK/POSTHOOK messages just like before; does this have a simple explanations?

Reply:
Minimr created a hdfs://tmp directory that had permissions 777. The first create table in this test creates a table in /tmp. That succeeded in mr because tmp is writable. Tez cluster does not have this tmp directory. The test creates a tmp directory itself but this is not created with 777 permissions. So the test fails in the first create table. I added a dfs chmod 777 to the test.